tests/storage/test_wffc.py

```
@@ -204,123 +180,67 @@ def test_wffc_create_vm_from_uploaded_dv_via_virtctl(
-        storage_class=data_volume_multi_wffc_storage_scope_module.storage_class,
-        consume_wffc=True,
-    ) as cdv:
-        cdv.wait_for_dv_success(timeout=TIMEOUT_4MIN)
-        create_vm_from_dv(client=unprivileged_client, dv=cdv, vm_name=cdv.name)
-
-
-@pytest.mark.sno
+@pytest.mark.polarion("CNV-4742")
 @pytest.mark.parametrize(
-    "data_volume_multi_wffc_storage_scope_function",
+    "rhel10_data_source_scope_module,blank_dv_wffc_scope_function",
```

Ruff (0.15.10)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)